usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1744,6 +1778,8 @@ CK_RV token_specific_ec_generate_keypair(STDLL_TokData_t * tokdata,
     if (return_code != CCA_SUCCESS) {
         TRACE_ERROR("CSNDPKB (EC KEY TOKEN BUILD) failed. return:%ld,"
                     " reason:%ld\n", return_code, reason_code);
+        if (is_curve_error(return_code, reason_code))
```

Comment:
Curve support is checked before the operation. How can the operation fail with curve not supported ?